verify-allowed-origin-deprecation.test.ts の type-check でエラーが発生

#160 より
GitHub Actions の type-check でエラーが発生しました。
type-check 実行時に、verify-allowed-origin-deprecation.test.ts に関連する部分でエラーが発生しています。

**エラー内容**
```

##***error***src/content-attestation/verify-allowed-origin-deprecation.test.ts(19,5): error TS2322: Type 'Mock<*** (...data: any***): void; (message?: any, ...optionalParams: any***): void; ***>' is not assignable to type 'Mock<new (...args: unknown***) => unknown>'.
  Type 'Mock<*** (...data: any***): void; (message?: any, ...optionalParams: any***): void; ***>' is not assignable to type 'MockInstance<new (...args: unknown***) => unknown>'.
    Types of property 'mock' are incompatible.
      Type 'MockContext<*** (...data: any***): void; (message?: any, ...optionalParams: any***): void; ***>' is not assignable to type 'MockContext<new (...args: unknown***) => unknown>'.
        Type '*** (...data: any***): void; (message?: any, ...optionalParams: any***): void; ***' is not assignable to type 'new (...args: unknown***) => unknown'.
          Type '*** (...data: any***): void; (message?: any, ...optionalParams: any***): void; ***' provides no match for the signature 'new (...args: unknown***): unknown'.
 ELIFECYCLE  Command failed with exit code 2.
```

--- a/packages/verify/src/content-attestation/verify-allowed-origin-deprecation.test.ts
+++ b/packages/verify/src/content-attestation/verify-allowed-origin-deprecation.test.ts
@@ -2,7 +2,15 @@ import { generateKey, LocalKeys } from "@originator-profile/cryptography";
 import { ContentAttestation } from "@originator-profile/model";
 import { signJwtVc } from "@originator-profile/securing-mechanism";
 import { addYears, fromUnixTime, getUnixTime } from "date-fns";
-import { afterEach, beforeEach, describe, expect, test, vi } from "vitest";
+import {
+  afterEach,
+  beforeEach,
+  describe,
+  expect,
+  test,
+  vi,
+  MockInstance,
+} from "vitest";
 import { opId } from "../helper";
 import { CaVerifier } from "./verify-content-attestation";
 
@@ -13,7 +21,7 @@ const caIssuer = opId.originator;
 
 describe("allowedOrigin deprecation 警告", async () => {
   const issuer = await generateKey();
-  let consoleWarnSpy: ReturnType<typeof vi.spyOn>;
+  let consoleWarnSpy: MockInstance<typeof console.warn>;
 
   beforeEach(() => {
     consoleWarnSpy = vi.spyOn(console, "warn").mockImplementation(() => {});

